CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 Added:
  * Allow `chronosphere_notification_policy` resources without a `team_id`.
+ * Reject unnamed `chronosphere_notification_policy` resources that set a `slug`
```

Comment:
was this already the case prior to the above change? if so, i think we can drop this, and add a `skip-changelog` label to the PR to skip the changelog check